packages/@aws-cdk/cfnspec/spec-source/specification/000_cfn/000_official/000_AWS_Organizations.json

```diff
@@ -0,0 +1,118 @@
+{
+  "$version": "97.0.0",
+  "PropertyTypes": {},
+  "ResourceTypes": {
```

Comment:
`AWS::Organizations::OrganizationalUnit` is missing: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-organizations-organizationalunit.html

Reply:
in v98: https://github.com/aws/aws-cdk/blob/e5a77e813bf4010a75a510355ccee563d3ed02c6/packages/%40aws-cdk/cfnspec/spec-source/specification/000_cfn/000_official/000_AWS_Organizations.json#L61